targets/unity-v2/source/Shared/Internal/OneDsUtility.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Net.Http;
+
+namespace PlayFab.Internal
+{
+    public static class OneDsUtility
+    {
+        // public const string ONEDS_SERVICE_URL = "https://mobile.events.data.microsoft.com/OneCollector/1.0/";
+        public const string ONEDS_SERVICE_URL = "https://self.events.data.microsoft.com/OneCollector/1.0/";
+        private static readonly long TICKS_AT_1970 = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc).Ticks;
+
+        public static long MsFrom1970()
+        {
+            return (DateTime.UtcNow.Ticks - TICKS_AT_1970) / 10000;
+        }
+        
+        public static void ParseResponse(long httpCode, Func<string> getText, string errorString, Action<object> callback)
+        {
+            if (!string.IsNullOrEmpty(errorString))
+            {
+                callback.Invoke(new PlayFabError
+                {
+                    Error = PlayFabErrorCode.Unknown,
+                    ErrorMessage = errorString
+                });
+            }
+            else
+            {
+                string httpResponseString;
+                try
+                {
+                    httpResponseString = getText.Invoke();
+                }
+                catch (HttpRequestException exception)
+                {
+                    var error = new PlayFabError();
+                    error.Error = PlayFabErrorCode.ConnectionError;
+                    error.ErrorMessage = exception.InnerException.Message;
+                    callback?.Invoke(error);
+                    return;
+                }
```

Comment:
Please remove this, the catch-all Exception block below should be sufficient. HttpRequestException is not supported in Unity 2017 (with 4.6 runtime) out-of-the-box, so we can remove this part without sacrificing much.